osmo-test-5/osmosis.zone_assets.json

```diff
@@ -54,6 +54,11 @@
       "chain_name": "chain4energytestnet",
       "base_denom": "uc4e",
       "path": "transfer/channel-111/uc4e"
+    },
+    {
+      "chain_name": "composabletestnet",
+      "base_denom": "ppica",
```

Comment:
I see here base denom 'ppica'. Just want to make sure 'ubanksy' is definitely not the base asset? Can we please correct the [assetlist.json](https://github.com/cosmos/chain-registry/blob/master/testnets/composabletestnet/assetlist.json) at the Chain Registry first?

Reply:
hi @JeremyParish69 yes the base denom is 'ppica' now, have updated it in the assetlist.json on chain registry. Thanks!